Gruntfile.js

```diff
@@ -1,5 +1,6 @@
 /*global module:false*/
 var _ = require('lodash');
+var config = require('./config');
```

Comment:
heroku doesn't use config.json but passes variables through the environment so i would check for `process.env.NODE_ENV == 'production'`